LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/IoTHubConnector.cs

```diff
@@ -151,10 +151,17 @@ private void CreateDeviceClient()
                 {
                     string partConnection = createIoTHubConnectionString();
                     string deviceConnectionStr = $"{partConnection}DeviceId={DevEUI};SharedAccessKey={PrimaryKey}";
-
-                    deviceClient = DeviceClient.CreateFromConnectionString(deviceConnectionStr, TransportType.Amqp_Tcp_Only);
-
-                   
+                    
+                    if (configuration.LeafDeviceProtocol.ToUpper() == "AMQP_TCP_ONLY")
+                    {
+                        deviceClient = DeviceClient.CreateFromConnectionString(deviceConnectionStr, TransportType.Amqp_Tcp_Only);
+                        Logger.Log(DevEUI, $"device client created using Amqp_Tcp_Only", Logger.LoggingLevel.Info);
+                    }
```

Comment:
I would log as full not as info because is not useful for non developers

Reply:
Instead of making upper and comparing use string.Equals(configuration.LeafDeviceProtocol, "amqp_tcp_only", StringComparison.InvariantCultureIgnoreCase).
It avoids creating a new instance of the string.